### PR TITLE
OpenNebulaEndpoint introduced

### PR DESCRIPTION
**Pull request by IgorPelevanyuk**
Tested on DIRAC@JINR and JINR Cloud OpenNebula.

### CODE REVIEW

File: VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
+###########################################################
+# $HeadURL$
+# File: OcciEndpoint.py
+# Author: A.T.
+#bbaac
+###########################################################
```
Comment (fstagni) [inline VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py]: Remove the lines above

File: VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+"""
+   OpenNebula Endpoint base class implementation for the OpenNebula cloud service.
+"""
+
+__RCSID__ = 'e069e3a (2018-05-15 22:19:34 +0200) Andrei Tsaregorodtsev <atsareg@in2p3.fr>'
```
Comment (fstagni) [inline VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py]: just put
```
__RCSID__ = "$Id"
```

File: VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+import xmlrpclib
+import ssl
+
+# DIRAC
+from DIRAC import gLogger, S_OK, S_ERROR
+from VMDIRAC.Resources.Cloud.Endpoint import Endpoint
+from VMDIRAC.Resources.Cloud.KeystoneClient import KeystoneClient
+from DIRAC.Core.Utilities.File import makeGuid
+
+DEBUG = True
```
Comment (fstagni) [inline VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py]: Why?

File: VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py [hunk 4]
```diff
@@ -0,0 +1,287 @@
+###########################################################
```
Comment (fstagni) [inline VMDIRAC/Resources/Cloud/OpenNebulaEndpoint.py]: Please use autopep8 on this file

**Comment by IgorPelevanyuk** [comment]
Corrected. And tested again on Cloud in JINR.

**Pull request merged**